src/main/java/by/custom/utilcalculator/directory/resources/Prices.java

```diff
@@ -1,4 +1,4 @@
-package by.custom.utilcalculator.repository.resources;
+package by.custom.utilcalculator.directory.resources;
 
 public class Prices {
```

Comment:
It's more common practice to use singular name for enum 
```suggestion
public class Price {
```

Reply:
Fixed.